apps/client/src/config/index.ts

```diff
@@ -1,4 +1,4 @@
-type Topic = {
+export type Topic = {
```

Comment:
Why do we export topic here? I don't see it imported anywhere

Reply:
Good point. Fixed